com.unity.shadergraph/Editor/GraphUI/ShaderGraphOnboardingProvider.cs

```diff
@@ -4,7 +4,7 @@
 using UnityEngine.GraphToolsFoundation.CommandStateObserver;
 using UnityEngine.UIElements;
 
-namespace UnityEditor.ShaderGraph.GraphUI.GraphElements.Windows
+namespace UnityEditor.ShaderGraph.GraphUI
 {
     public class ShaderGraphOnboardingProvider : OnboardingProvider
```

Comment:
Does the onboarding provider still prompt to create an asset? if we could disable that for now it would be nice. But that can be a separate PR.

Reply:
It does. I'll get that in a different PR.

Reply:
I made a jira task for it https://jira.unity3d.com/browse/GSG-888